lib/install/phases/bootstrap.go

```diff
@@ -363,6 +371,13 @@ func (p *bootstrapExecutor) configureServiceUser() error {
 	return nil
 }
 
+func (p *bootstrapExecutor) isMonitoringEnabled() bool {
+	if p.Application.Manifest.Extensions != nil && p.Application.Manifest.Extensions.Monitoring != nil && p.Application.Manifest.Extensions.Monitoring.Disabled {
```

Comment:
To be honest, I had the same question as @a-palchikov. Why not simplify things and just always create it? /var/lib/gravity is our state directory and we control what we put in it, users should not be concerned with what directories we need there.

Reply:
Ok, changed it to be always created.